src/dso_api/dynamic_api/views/api.py

```diff
@@ -105,7 +105,7 @@ def lookup_field(self):
         """Overwritten from GenericAPIView to filter on temporal identifier instead."""
         if self.temporal.is_versioned:
             # Take the first field as grouper from ["identificatie", "volgnummer"]
-            return self.model.table_schema().identifier[0]
+            return self.model.table_schema().identifier_fields[0].db_name
```

Comment:
@jasper-jager zeker weten dat niet niet `.python_name` moet zijn? We gebruiken dat veld immers in het ORM..

Reply:
Goede, ik was eigenlijk niet zo bewust van de `.python_name`, maar de `.db_name` wordt mogelijk afgekort natuurlijk. Ik zal het nog even aanpassen